docs/setup.md

```diff
@@ -82,7 +82,8 @@
 3. **Start Gulp to watch JS and CSS**
 
       $ `gulp`
-
+      - if you are getting error message "Error: EACCES: permission denied, open '/home/rlot/.babel.json'", use:
+      `BABEL_DISABLE_CACHE=1 gulp`
```

Comment:
Also, put in 'username' or the like in place of your own username in the example error message.